### DISCUSSION TITLE
Testing each Pull Request with an automated full stack deployment, one URL per branch

### DISCUSSION THREAD

**Discussion by emiquelito** [joomla/joomla-cms#37414] [General]
Sorry if this is not the best place to post this question. This is related to

https://github.com/joomla/joomla-cms/issues/32495

Are you currently happy with your process of testing each Pull Request from an end user perspective? would it be helpful to have an automated way for commissioning full stack deployments so you can test each branch in isolation (including on different PHP or MySQL versions, etc)?

Lots have changed on our end and I'd be happy to offer unlimited environments to support Joomla, we love Open Source and would like to give back to the community.

We have used Joomla in the past and we would be happy to provide integration samples (Squash uses a standard YAML file) so you don't have to spend time on this.

Thanks

---

Hey @Hackwar  and @Llewellynvdm I hope all is well, it's been a while since our last discussion. Let me know if you have any thoughts.

**Comment by Hackwar** [comment]
~~Hello @emiquelito, thank you for getting into contact with us again. Unfortunately, I don't think we will be using your companies services with Joomla in the forseeable future.~~

**Comment by Hackwar** [comment]
I brought this up in the maintainer chat and have to apologise for my earlier response. My answer was based on our last interaction and I didn't give this enough attention. How would this work? If we only need to a add a yaml to our repo, I would ask you to open a PR with such a file and mention me in there, so that we can discuss this further there. (Base the PR on the 4.1-dev branch for now) Otherwise please contact me by mail at hannes.papenberg@community.joomla.org

**Reply by emiquelito** [reply]
Thanks @Hackwar. I sent you an email yesterday with some additional info. We will be submitting a PR very soon.

Thanks again.
